sdcard/rootfs/kube-systemd/usr/lib/systemd/system/k8s-master.service

```diff
@@ -17,6 +17,7 @@ ExecStart=/bin/sh -c "exec docker run \
         -v /var/lib/docker/:/var/lib/docker:rw \
         -v /var/lib/kubelet:/var/lib/kubelet:shared \
         -v /var/run:/var/run:rw \
+        -v /var/log/containers:/var/log/containers:rw \
```

Comment:
Docker places the `*-json.log` file in `/var/lib/docker/containers/<container_id>/`
The `kubelet` creates a link to the container log file in `/var/log/containers`.

That's why the `kubelet` needs access to `/var/log/containers` so that it can create the link.
Example:
`k8s-master-192.168.1.30_default_scheduler-9acaf0821554295896df64c616431733049ac1fbf340b73234ba06ff66f88f1f.log`